### ISSUE TITLE
ImportWarning due to ambiguous 'ssl' directory

### ISSUE THREAD

**Issue by hheimbuerger** [chargebee/chargebee-python#29]
Importing v2.6.1 on Python 2.7 with deprecation warnings enabled raises an ImportWarning:

```
henrik@debian:~$ python -Wall -c "import chargebee"
/local/lib/python2.7/site-packages/chargebee/compat.py:40: ImportWarning: Not importing directory '/local/lib/python2.7/site-packages/chargebee/ssl': missing __init__.py
  import ssl
```

**Comment by cb-rakesh** [comment]
Let me know if the issue persists with the latest version as well. Or else we can close this.

**Comment by hheimbuerger** [comment]
That was years ago and I'm not anymore involved at that level.

I did a quick code review and I don't see an `import ssl` anywhere in your code anymore, so it probably should be fine. I still think it's an unfortunate/risky naming choice for a data directory in your Python source tree — why don't you just call it `certificates`? Less of a risk of a naming conflict, and much more accurate anyway, given that [we've phased out the SSL protocol years ago](https://en.wikipedia.org/wiki/Transport_Layer_Security#SSL_1.0,_2.0,_and_3.0).

If you tested on Python 2.7 and couldn't reproduce this, then go ahead and close my issue.
